code/game/gamemodes/changeling/powers/lesserform.dm

```diff
@@ -12,9 +12,7 @@
 		return 0
 	user << "<span class='warning'>Our genes cry out!</span>"
 
-	user.monkeyize(TR_KEEPITEMS | TR_KEEPIMPLANTS | TR_KEEPORGANS | TR_KEEPDAMAGE | TR_KEEPSE | TR_KEEPSRC)
+	user.monkeyize(TR_KEEPITEMS | TR_KEEPIMPLANTS | TR_KEEPORGANS | TR_KEEPDAMAGE | TR_KEEPVIRUS | TR_KEEPSE)

```

Comment:
Changeling lesser form / human form doesn't get you rid of viruses anymore (that's anatomic panacea's job).